perf/benchmark/templates/fortio.yaml

```diff
@@ -394,6 +394,11 @@ spec:
       volumes:
       - name: shared-data
         emptyDir: {}
+      {{- if and (eq $.name "fortioclient") ($.Values.cert.server) }}
+      - name: gateway-secret
+        secret:
+          secretName: fortio-server-ingress-cert
```

Comment:
Don't mount, use credentialName in `Gateway`. Mounting is the legacy way

Reply:
If I understand your comment correctly, I think I'm already doing that here:
https://github.com/istio/tools/blob/f1c4203fe9fc8382e0fb8eedd1af591026b2b344/perf/benchmark/templates/fortio.yaml#L602-L612

I mount the secret to use it with Fortio's `-cacert` flag. The parameter for Fortio is given here:
https://github.com/istio/tools/blob/f1c4203fe9fc8382e0fb8eedd1af591026b2b344/perf/benchmark/configs/istio/ingressgateway_tls/cpu_mem.yaml#L25

I left an old configuration which I think uses the mount-based approach with a comment, should I just remove it?
https://github.com/istio/tools/blob/f1c4203fe9fc8382e0fb8eedd1af591026b2b344/perf/benchmark/templates/fortio.yaml#L589-L601

Reply:
ohh sorry, misunderstood this was for fortio. Fine to mount for fortio, for all istio things we should use credentialName

Reply:
Is this good to go? @howardjohn